cep-jlap.md

```diff
@@ -0,0 +1,247 @@
+<table>
+<tr><td> Title </td><td> JSON Patch Incremental Updates for repodata.json </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Daniel Holth &lt;dholth@gmail.com | dholth@anaconda.com&gt;</td></tr>
+<tr><td> Created </td><td> Mar 30, 2022</td></tr>
+<tr><td> Updated </td><td> Oct 11, 2022</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> https://github.com/dholth/repodata-fly </td></tr>
```

Comment:
Might as well link to the implementations in conda and librattler. conda's jlap core is pretty elegant. The network error handling less so.